### DISCUSSION TITLE
Documentation UI/UX: Sub-Header and Document Title

### DISCUSSION THREAD

**Discussion by philleepflorence** [directus/directus#3720] [Feature Requests]
## What 

Include subheader to indicate the section of Documentation as well as update the Title Metatag Template.

## Why

Improve Discovery - See [Documentation: A Discovery Summary and Guide.](https://github.com/directus/directus/discussions/4326)  
Add context to the current page.

## How

Add sub-header under the page H1 tag.
Update the template for the Document Title Metatag to include the section, such as _Getting Started_, _Concept_, et al.

## Where

Directus Admin Application - Documentation.
Directus Documentation Website.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
